rules/cloud/gcp/audit/gcp_context_aware_access_levels_deleted.yml

```diff
@@ -0,0 +1,26 @@
+title: GCP Context Aware Access Levels Deleted
+id: d8007af7-664a-494e-a1db-848a518611c5
+description: |
+  Detects when a context aware access level is deleted. A context aware access level is part of BeyondCorp Enterprise which is Google's way of enforcing Zero Trust model.
+  An adversary would be able to remove access levels to gain easier access to google workspace resources.
+references:
+    - https://cloud.google.com/access-context-manager/docs/reference/rest/v1/accessPolicies.accessLevels
+    - https://support.google.com/a/answer/12645308
+author: Bryan Lim
+date: 2023/09/12
+status: experimental
+tags:
+  - attack.persistence
+  - attack.t1098.003
+logsource:
+  product: gcp
+  service: gcp.audit
+detection:
+  selection:
+    data.protoPayload.authorizationInfo.permission: accesscontextmanager.accessLevels.delete
```

Comment:
From the docs https://cloud.google.com/access-context-manager/docs/audit-logging this value should `accesscontextmanager.accessPolicies.accessLevels.delete` as its part of the Access Context Manager operations. But as it's covered in your rule ` GCP Access Policy Deleted` its a duplicate. So it'll be deleted.